src/components/slider/slider.spec.js

```diff
@@ -23,13 +23,30 @@ describe('md-slider', function() {
       'getBoundingClientRect'
     ).and.returnValue(angular.extend({
       width: 100,
+      height: 100,
```

Comment:
I am pretty sure you do not need width/height if you set the top/bottom/left/right all to `0`. Perhaps we can remove it?